libs/utils/src/lib/format/number.spec.ts

```diff
@@ -23,6 +24,21 @@ describe('number utils', () => {
     }
   );
 
+  it.each([
+    { v: new BigNumber(123000), d: 5, o: '1.23', q: 0.1 },
+    { v: new BigNumber(123000), d: 3, o: '123.00', q: 0.1 },
+    { v: new BigNumber(123000), d: 1, o: '12,300.00', q: 0.1 },
+    { v: new BigNumber(123001), d: 2, o: '1,230.01', q: 0.1 },
+    { v: new BigNumber(123001000), d: 2, o: '1,230,010.00', q: 0.1 },
```

Comment:
**comment (non-blocking)**: I copied in David for a review of these but they *look* good to me.